### PR TITLE
Adding a docker compose script.

### PR DESCRIPTION
**Pull request by JeremyLWright**
This allows one to open their notebook in the docker container, but the
notebook is "delegated" through the container. Hence, you can edit the
notebook and your changes will persist outside the container.

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -6,9 +6,6 @@ ENV NB_USER ${NB_USER}
 ENV NB_UID ${NB_UID}
 RUN adduser --uid ${NB_UID} ${NB_USER}
 
-COPY *.ipynb /home/${NB_USER}/
-RUN chown -R ${NB_USER} /home/${NB_USER}
```
Comment (Alexander-N) [inline Dockerfile]: I think these lines have to stay or the notebook won't run in Binder.

**Review by Alexander-N — CHANGES_REQUESTED** [review]
Nice, that makes local installation a lot easier. The main reason for me to add the Dockerfile was to make it runnable in [Binder](https://mybinder.org/), so we should make the change in a way that this will still work, maybe add a separate Dockerfile?

File: Dockerfile (hunk 1 above)
Comment (JeremyLWright) [reply Dockerfile]: Understood. Done. :-)

**Comment by Alexander-N** [comment]
Thanks!

**Pull request merged**